Schemas/UserSchema.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+let Schema = mongoose.Schema;
+
+let UserSchema = new Schema({
+    userName: {type: String, required: true},
```

Comment:
if the schema is for users - no need to prefix names of all fields with 'user'.

Reply:
fix on branch form-validation